src/build.py

```diff
@@ -1397,7 +1406,7 @@ def TestBare():
       wasmjs=os.path.join(INSTALL_LIB, 'wasm.js'),
       extra_files=[os.path.join(INSTALL_LIB, 'musl.wasm')])
 
-  if IsMac():
+  if IsMac() and 'JSC' not in buildbot.WarnedList():
```

Comment:
It feels kind of unfortunate to me that this line has to care whether JSC failing is a `Fail` or a `Warn`. Maybe `and not buildbot.DidStepFail('JSC')`, where `DidStepFail` checks the warning and failure arrays?